twitter/API/APIManager.m

```diff
@@ -65,6 +65,16 @@ - (void)getHomeTimelineWithCompletion:(void(^)(NSMutableArray *tweets, NSError *
        completion(nil, error);
    }];
 }
+- (void)postStatusWithText:(NSString *)text completion:(void (^)(tweet *, NSError *))completion{
+    NSString *urlString = @"1.1/statuses/update.json";
+    NSDictionary *parameters = @{@"status": text};
+    [self POST:urlString parameters:parameters progress:nil success:^(NSURLSessionDataTask * _Nonnull task, NSDictionary *  _Nullable tweetDictionary) {
+        tweet *newTweet = [[tweet alloc]initWithDictionary:tweetDictionary];
+        completion(newTweet, nil);
```

Comment:
Do you need to make sure tweet was successfully created?

Reply:
Oh Yes! edge cases I will do it